docs/UserGuide.md

```diff
@@ -185,89 +188,93 @@ Format: `help`
 
-### Locating persons by name: `find`
+> **Note:** 
+> * At least one of the optional fields must be provided. (i.e NAME, PHONE, EMAIL, ADDRESS)
+> * Editing tags overwrites existing tags; it's not cumulative. To remove all tags, use `t/` without specifying any
+> tags after it.
+> * You can remove all the person’s tags by typing `t/` without
+      specifying any tags after it.
 
-Finds persons whose names contain any of the given keywords.
+### Locating Persons by Name: `find`
 
-Format: `find KEYWORD [MORE_KEYWORDS]`
+You can find a person whose name contain any of the given keywords.
 
-* The search is case-insensitive. e.g `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only the name is searched.
-* Only full words will be matched e.g. `Han` will not match `Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search).
-  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+Format: `find KEYWORD [MORE_KEYWORDS]`
 
 Examples:
 * `find John` returns `john` and `John Doe`
 * `find alex david` returns `Alex Yeoh`, `David Li`<br>
   ![result for 'find alex david'](images/findAlexDavidResult.png)
 
+> **Note:**
+> * The search is case-insensitive. e.g `hans` will match `Hans`
+> * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
+> * You can remove all the person’s tags by typing `t/` without
+    specifying any tags after it.
+> * Only full words will be matched e.g. `Han` will not match `Hans`
+> * Persons matching at least one keyword will be returned (i.e. `OR` search). e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+
 ### Deleting a person : `delete`
 
-Deletes the specified person from the address book.
+You can delete any specific person from your address book.
 
 Format: `delete INDEX`
 
-* Deletes the person at the specified `INDEX`.
-* The index refers to the index number shown in the displayed person list.
-* The index **must be a positive integer** 1, 2, 3, …​
-
 Examples:
 * `list` followed by `delete 2` deletes the 2nd person in the address book.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
+> **Note:**
+> * Deletes the person at the specified `INDEX`.
+> * The index refers to the index number shown in the displayed person list.
+> * The index **must be a positive integer** 1, 2, 3, …​
+
 ### Clearing all entries : `clear`
 
-Clears all entries from the address book.
+You can clear all entries from your address book.
 
 Format: `clear`
 
 ### Exiting the program : `exit`
 
-Exits the program.
+You can exit the program.
 
 Format: `exit`
 
 ### Saving the data
 
-F.A.K.E.J.A.R.V.I.S. data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+No need to worry about saving. F.A.K.E.J.A.R.V.I.S. data are saved in the hard disk automatically after any command that changes the data.
```

Comment:
I think this is too informal

Reply:
Woops, forget to remove this part. thanks eagle eye